bindings/python/pinocchio/utils.py

```diff
@@ -27,12 +27,6 @@ def skew(p):
     x, y, z = p
     return np.matrix([[0, -z, y], [z, 0, -x], [-y, x, 0]], np.double)
 
-
-se3ToXYZQUAT = pin.se3ToXYZQUAT
```

Comment:
I added them in #720 for backward compatibility. Shouldn't we keep them ?

By the way,  `se3ToXYZQUAT = pin.se3ToXYZQUAT` should have been  `se3ToXYZQUAT = pin.se3ToXYZQUATtuple`

Reply:
I think they should be kept as deprecated

Reply:
They are already exposed in the main scope, but we can indeed retrieve the previous behavior

Reply:
with deprecation.

Reply:
> They are already exposed in the main scope

Yes, but in many examples/tutorials, I see
```python
from pinocchio.utils import *
import pinocchio as pin
```
In this case, they become unavailable.

A more concise declaration would be
```python
from pinocchio import se3ToXYZQUATtuple as se3ToXYZQUAT, XYZQUATToSe3
```
but then, they wouldn't be deprecated.